tools/azure-sdk-tools/devtools_testutils/azure_testcase.py

```diff
@@ -182,34 +187,52 @@ def setUp(self):
     def tearDown(self):
         return super(AzureTestCase, self).tearDown()
 
-    def create_basic_client(self, client_class, **kwargs):
 
-        tenant_id = os.environ.get("AZURE_TENANT_ID", None)
-        client_id = os.environ.get("AZURE_CLIENT_ID", None)
-        secret = os.environ.get("AZURE_CLIENT_SECRET", None)
 
-        if tenant_id and client_id and secret and self.is_live:
+    def _create_credential(self, client_secret_credential):
```

Comment:
I'm not sure if this deserves a single method, as it simply constructs the credential and returns

Reply:
+1